packages/react/src/SubNav/SubNav.tsx

```diff
@@ -225,19 +384,27 @@ const SubNavLinkWithSubmenu = forwardRef<HTMLDivElement, SubNavLinkProps>(
           aria-current={ariaCurrent}
           {...props}
         >
-          <Text as="span" size="200" className={styles['SubNav__link-label']}>
+          <Text
+            as="span"
+            size="200"
+            className={styles['SubNav__link-label']}
+            variant={ariaCurrent === 'page' ? 'default' : 'muted'}
+          >
             {label}
           </Text>
         </a>
-        <button
-          className={styles['SubNav__sub-menu-toggle']}
-          onKeyDown={onKeyDown}
-          aria-expanded={isExpanded ? 'true' : 'false'}
-          aria-controls={submenuId}
-          aria-label={`${isExpanded ? 'Close' : 'Open'} submenu`}
-        >
-          <ChevronDownIcon className={styles['SubNav__sub-menu-icon']} size={16} />
-        </button>
+        {isLarge && (
+          <button
+            className={styles['SubNav__sub-menu-toggle']}
+            onKeyDown={onKeyDown}
```

Comment:
Unsure why, but this doesn't seem to be working for me in Chrome on Windows. Hitting space or Enter while the button is focussed doesn't do anything.

Happy to do a bit more digging, but I wanted to flag now so I didn't forget.

Reply:
Ah ok, this is a new one for me. The `onKeyDown` is working on Windows, however the screen reader I'm using (NVDA) is eating the keypresses when in browse mode. If I switch to focus mode (the mode used for interacting with the page) then it works as expected.

It could be easy to dismiss this as unintuitive screen-reader behaviour, but from a user point of view they just hear that they're focussing a button (_"Open submenu button collapsed"_) but they're unable to open it using space or Enter, which should work for a button.

I'm guessing they open/close behaviour is bound to keydown here to prevent users from being able to click on the dropdown button? My initial feeling is that the simplest way forward would be to remove that logic and treat this like a normal button and bind the open/close behaviour to a click instead.

wdyt?